clients/src/main/java/org/apache/kafka/common/record/Records.java

```diff
@@ -90,4 +91,18 @@ public interface Records extends TransferableRecords {
      * @return The record iterator
      */
     Iterable<Record> records();
+
+    /**
+     * Return a slice of records from this instance, which is a view into this set starting from the given position
```

Comment:
We are duplicating the same javadoc in 3 places. We could just keep it once in Records.

Reply:
Agree, done.